account_payment_order/models/account_move.py

```diff
@@ -207,3 +214,23 @@ def create_account_payment_line(self):
                 }
             )
         return action
+
+    def action_payment_lines(self):
+        self.ensure_one()
```

Comment:
Is 'self.ensure_one()' necessary? From my point of view, a future/possible improvement or an open door at the level of a particular project would be an action in the list view for a selection of invoices.